tools/ci_build/github/apple/test_ios_packages.py

```diff
@@ -84,6 +105,10 @@ def parse_args():
                         help='This script will fail if CocoaPods is not installed, '
                         'will not throw error unless fail_if_cocoapod_missing is set.')
 
+    parser.add_argument("--framework-info-file", type=pathlib.Path, required=True,
```

Comment:
>framework-info-file [](http://example.com/codeflow?start=27&length=19)

nit: within the same script, could we use the same option word delimiters? e.g., --framework_info_file